src/__tests__/__snapshots__/render.test.tsx.snap

```diff
@@ -124,6 +124,224 @@ exports[`debug changing component: bananaFresh button message should now be "fre
 </View>"
 `;
 
+exports[`debug should use debugOptions from config when no option is specified 1`] = `
+"<View>
+  <Text>
+    hello
+  </Text>
+</View>"
+`;
+
+exports[`debug should use given options over config debugOptions 1`] = `
+"<View
+  style={
+    Object {
+      "backgroundColor": "red",
+    }
+  }
+>
+  <Text>
+    hello
+  </Text>
+</View>"
+`;
+
+exports[`debug with all props filtered 1`] = `
+"<View>
+  <Text>
+    Is the banana fresh?
```

Comment:
Shouldn't `children` be filtered also?

Reply:
I think this is due to `children` prop being extracted from `props` in both TestInstance and JSON. Imo we should keep this as is, as a reasonable behaviour for most cases.

Reply:
I was surprised by this as well but I actually think that this is fine, we'll want to keep the children prop in most cases and not having to explicitly include it is nice